### PR TITLE
fixing side nav display issue

### PR DESCRIPTION
**Pull request by danid123**
Fixed the menu display issue. I'm going to work on adding the sidebar menu to the mobile menu.

### CODE REVIEW

**Comment by mgifford** [comment]
Just tested it here https://federalist-6d3494a6-5b03-42fa-82f2-57f3bbb203af.app.cloud.gov/preview/gsa/openacr-website/menu-layout-fix/about/

Looks great.

Screen Shot of sidebar menu working

File: _layouts/page.html [hunk 1]
```diff
@@ -9,11 +9,9 @@
 <div id="main-content" class="usa-layout-docs usa-section">
   <div class="grid-container">
     <div class="grid-row grid-gap">
-      {% if page.sidenav == true %}
       {% include sidenav.html %}
```
Comment (mgifford) [inline _layouts/page.html:12]: I think that we'll want to have this conditional left in. It is something set in the page.md files and I don't see any reason to remove it. Although maybe I'm missing something. 

```
      {% if page.sidenav == true %}
       {% include sidenav.html %}
       {% endif %}
```

Comment (danid123) [reply _layouts/page.html]: I added the conditionals back

**Review by mgifford — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**